contributing/code/bc.rst

```diff
@@ -32,6 +32,14 @@ The second section, "Working on Symfony Code", is targeted at Symfony
 contributors. This section lists detailed rules that every contributor needs to
 follow to ensure smooth upgrades for our users.
 
+.. warning::
+
+    Code marked with the ``@experimental`` or the ``@internal`` tags are
+    excluded from our Backwards Compatibility promise.
+
+    Also note that backward compatibility breaks are tolerated if they help fix
```

Comment:
Your reference seems to state `backward` without the trailing `s` is the most appropriate as an adjective:

> If you’re using British English, it’s typically backwards as an adverb and backward as an adjective, so you have two things to remember.

So maybe the BC promise doc should be updated instead 😄